### ISSUE TITLE
Abstract over model (pytorch vs keras)

### ISSUE THREAD

**Issue by mikekestemont** [hipster-philology/pandora#11]
Things needed for abstracting over the Keras vs Pytorch API:

- serialization: save and load model to dir (including model definition and weights)
- model.compile() should move to model*.py
- model.predict()
- get weights for the embeddings (not essential: mainly for viz)
- adjust learning rate
- model.fit()

**Comment by mikekestemont** [comment]
I created the new pytorch branch and implemented the model abstraction there + added more documentation. It should now be possible for @emanjavacas to provide a PyTorch model on top of the Keras (Theano/Tensforflow) model. Let me know if there are any issues that arise.

**Comment by emanjavacas** [comment]
I'll take over this, however I just realized that I don't seem to have push access. Perhaps due to the fact that I am still not a member of the pandora-team?

**Comment by PonteIneptique** [comment]
Fixed :)

**Comment by emanjavacas** [comment]
Thanks! :-)

**Comment by mikekestemont** [comment]
We can take a fresh start here!

**Comment by emanjavacas** [comment]
some thing to be done here?

**Comment by emanjavacas** [comment]
I believe this can be closed.
